### PR TITLE
NEW Auto notes

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripeltd/product-issues/issues/754

Example CI runs in issue description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,21 +2,38 @@
 
 Create a tag and an optional release
 
-## Usage
+### Usage
```
Comment (ChloeSartorelli) [inline README.md]:
```suggestion
## Usage
```
This is the first heading after the 1st level heading which makes it a 2nd level heading. Please also adjust the other headings below as appropriate.

File: README.md [hunk 2]
```diff
@@ -2,21 +2,38 @@
...
     with:
       tag: 1.2.3
       release: true
 ```
 
-Read more information about the inputs available for [silverstripe/gha-pull-request](https://github.com/silverstripe/gha-pull-request).
+#### Inputs
 
-## Why there is no SHA input paramater
+##### tag
```
Comment (ChloeSartorelli) [inline README.md]:
```suggestion
##### tag (required)
```

File: action.yml [hunk 3]
```diff
@@ -145,23 +150,27 @@ runs:
...
+        GITHUB_SHA: ${{ github.sha }}
       run: |
-        # Create new release via GitHub API
-        # https://docs.github.com/en/rest/releases/releases#create-a-release
         # Escape double quotes '"' => '\"'
         RELEASE_DESCRIPTION=${RELEASE_DESCRIPTION//\"/\\\"}
-        RESP_CODE=$(curl -w %{http_code} -s -o /dev/null \
+        # Create new release via GitHub API
+        # https://docs.github.com/en/rest/releases/releases#create-a-release
+        # TODO: change __response.json back to /dev/null
```
Comment (ChloeSartorelli) [inline action.yml]: TODO comment here - I'm guessing something you intended to do before marking ready for review?

File: action.yml [hunk 4]
```diff
@@ -145,23 +150,27 @@ runs:
...
           "tag_name": "$TAG",
-          "target_commitish": "${{ github.sha }}",
+          "target_commitish": "$GITHUB_SHA",
           "name": "$TAG",
           "body": "$RELEASE_DESCRIPTION",
           "draft": false,
-          "prerelease": false
+          "prerelease": false,
+          "generate_release_notes": ${{ inputs.release_auto_notes }},
+          "make_latest": "legacy"
```
Comment (ChloeSartorelli) [inline action.yml:172]: Just noting for if we look back at this in the future and wonder why this legacy change was made: 
"legacy specifies that the latest release should be determined based on the release creation date and higher semantic version."

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (emteknetnz) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (emteknetnz) [reply README.md]: Updated

File: action.yml (hunk 3 above)
Comment (emteknetnz) [reply action.yml]: Removed

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**